tests/src/tracing/common/Assert.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace Tracing.Tests.Common
+{
+    public static class Assert
+    {
+        public static void Equal<T>(T left, T right) where T : IEquatable<T>
+        {
+            if (!left.Equals(right))
```

Comment:
nit: You might want to special case when left is null. Null.Equals throws. Same for NotEqual.

Reply:
Thanks @xiaomi7732.  Fixed.